content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -5,7 +5,8 @@ your model registry, Iterative Studio creates an annotation for it in a
 `dvc.yaml` file in your Git repository. You can add a model in any of the
 following ways:
 
-1. Log your model during the training process using [dvclive].
+1. Log your model during the training process using [dvclive]. It is done by
+   calling [`log_artifact`] method.
```

Comment:
Should we specify that it should have `type=model` (or show an example like `live.log_artifact("path/to/model", type="model", name="mymodel")`)?